src/tribler/core/utilities/utilities.py

```diff
@@ -272,6 +272,27 @@ def to_fts_query(text):
     return ' '.join(words)
 
 
+def parse_bool(obj):
+    """
+    Parse input to boolean True or False
+    Allow parsing text 'false', 'true' '1', '0' to boolean
+
+    :param obj: Object to parse
+    """
+    if isinstance(obj, str):
+        if obj.lower() == "false":
+            return False
+        
+        if obj.isnumeric():
```

Comment:
According to [Python documentation for `str.isnumeric()`](https://docs.python.org/3/library/stdtypes.html#str.isnumeric),
> "Numeric characters include digit characters, and all characters that have the Unicode numeric value property, e.g., U+2155, VULGAR FRACTION ONE FIFTH"

That means that even if `obj.isnumeric()` evaluates to `True`, it is possible to get `ValueError` when calling `int(obj)`.